tests/test_venv.py

```diff
@@ -402,7 +402,7 @@ def test_install_python3(tmpdir, newmocksession):
     assert len(l) == 1
     args = l[0].args
     assert "pip" in args[0]
-    for x in args:
```

Comment:
I've did not dwindle for now in it's rightness, just fixed the inadequate look from style point of view, e.g. variable not used and not starting with underscore